docs/readthedocs/source/doc/LLM/Quickstart/install_windows_gpu.md

```diff
@@ -0,0 +1,95 @@
+# Install BigDL-LLM on Windows for Intel GPU
+
+## MTL & iGPU & Arc
```

Comment:
What do this line mean?

Reply:
Change it to a text line "This guide applies to Intel [Meteor Lake](https://en.wikipedia.org/wiki/Meteor_Lake) iGPU and Core 11 and 12 generation iGPU, as well as Intel Arc Series GPU". Is it okay?

Reply:
Core Ultra
Core 12 - 14 gen

Reply:
revised